consensus/core/src/core_thread.rs

```diff
@@ -5,18 +5,20 @@ use crate::block::{Block, BlockRef, Round};
 use crate::context::Context;
 use crate::core::Core;
 use crate::core_thread::CoreError::Shutdown;
-use mysten_metrics::monitored_scope;
+use mysten_metrics::{metered_channel, monitored_scope};
 use std::fmt::Debug;
 use std::sync::Arc;
 use std::{collections::HashSet, thread};
 use thiserror::Error;
+use tokio::sync::oneshot;
 use tokio::sync::oneshot::error::RecvError;
-use tokio::sync::{mpsc, oneshot};
 use tracing::warn;
 
+const CORE_THREAD_COMMANDS_CHANNEL_SIZE: usize = 32;
```

Comment:
ooc how is this number chosen? Do we not expect any messages to be queued in this channel as the core thread is mostly a pass through for the messages to other components?

Reply:
For now I've just used whatever we had on the original mysticeti implementation which was based on our so far testing. As you said in theory the underlying ops here should be quite fast so this doesn't block / queue up . We can tune after further experimentation